test/expected/extension.out

```diff
@@ -1,80 +1,219 @@
 -- This file and its contents are licensed under the Apache License 2.0.
 -- Please see the included NOTICE for copyright information and
 -- LICENSE-APACHE for a copy of the license.
-\c :TEST_DBNAME
---list all extension functions in public schema
-SELECT DISTINCT proname
-FROM pg_proc
-WHERE oid IN (
-    SELECT objid
-    FROM pg_catalog.pg_depend
-    WHERE refclassid = 'pg_catalog.pg_extension'::pg_catalog.regclass
-      AND refobjid = (
-        SELECT oid
-        FROM pg_extension
-        WHERE extname = 'timescaledb')
-        AND deptype = 'e'
-        AND classid = 'pg_catalog.pg_proc'::regclass)
-    AND pronamespace = 'public'::regnamespace
-    AND proname NOT LIKE '%telemetry%'
-  ORDER BY proname;
-              proname               
-------------------------------------
- add_compression_policy
- add_continuous_aggregate_policy
- add_data_node
- add_dimension
- add_job
- add_reorder_policy
- add_retention_policy
- alter_job
- approximate_row_count
- attach_data_node
- attach_tablespace
- chunk_compression_stats
- chunks_detailed_size
- compress_chunk
- create_distributed_hypertable
- create_distributed_restore_point
- create_hypertable
- decompress_chunk
- delete_data_node
- delete_job
- detach_data_node
- detach_tablespace
- detach_tablespaces
- distributed_exec
- drop_chunks
- first
- histogram
- hypertable_compression_stats
- hypertable_detailed_size
- hypertable_index_size
- hypertable_size
- interpolate
- last
- locf
- move_chunk
- recompress_chunk
- refresh_continuous_aggregate
- remove_compression_policy
- remove_continuous_aggregate_policy
- remove_reorder_policy
- remove_retention_policy
- reorder_chunk
- run_job
- set_adaptive_chunking
- set_chunk_time_interval
- set_integer_now_func
- set_number_partitions
- set_replication_factor
- show_chunks
- show_tablespaces
- time_bucket
- time_bucket_gapfill
- timescaledb_fdw_handler
- timescaledb_fdw_validator
- timescaledb_post_restore
- timescaledb_pre_restore
-(56 rows)
+\t
```

Comment:
```suggestion
\pset tuples_only on
```

It's up to you but I like to be a bit more verbose using those PSQL meta-commands. Otherwise LGTM.